api_helper.py

```diff
@@ -1,17 +1,99 @@
-import requests
+import time
 import logging
+import psycopg2
+import requests
 from psycopg2 import Error
+
+def log_time(func):
+    def wrapper(*args, **kwargs):
+        start_time = time.time()
+        result = func(*args, **kwargs)
+        end_time = time.time()
+        logging.info(f"{func.__name__} took {end_time - start_time} seconds to complete.")
+        return result
+    return wrapper
+
 class APIFetch:
```

Comment:
as this class contains functions like both API operations and database operations, change the name to appropriate

Reply:
I changed the class name to DataHandler

You can see below
![image](https://github.com/DGMahendra/api_project/assets/161186439/5348b5af-b351-41d9-9b6c-70ecd327b9eb)